lib/pharos/command.rb

```diff
@@ -12,8 +12,14 @@ def self.options(*option_names)
       end
     end
 
-    def run(*_args)
+    def parse(*_arguments)
```

Comment:
Instead of reimplementing `run` like in #1136 - override parse. Now we can just super in both.